.github/workflows/legend-release.yml

```diff
@@ -91,9 +233,9 @@ jobs:
         run: |
           artifact_group='org.finos.legend.shared'
           artifact_name='legend-shared'
-          artifact_version='${{ github.event.inputs.legend-shared-release-version }}'
+          artifact_version='${{ needs.compute-versions.outputs.legend_shared_release_version }}'
           
-          max_retries=20
+          max_retries=120
```

Comment:
I'll try to see if we can increase this limit. Unfortunately maven is not that reliable so it can take time for artifacts to appear. So its either we fail if we take more than 20 mins or be optimistic and move forward with a hope that even if takes longer we might finish within the limit. I don't see a harm in continuing as if don't continue waiting we anyway fail.
But will try to see if I can bump the limit

Reply:
Since the limit is per job as you mentioned (https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) I think it's fine. If we push engine to more than 4 hours (worst case maven takes 2hrs) I feel we should rather look at engine build. Also this is the worst case and other alternative is to fail early which doesn't help either. So I would rather go with increased timeout